pkg/operator/operator.go

```diff
@@ -707,18 +704,20 @@ func (o *Operator) sync(ctx context.Context, key string) error {
 	return nil
 }
 
-func (o *Operator) reportError(ctx context.Context, err error, failedTaskReason string) {
-	klog.Infof("ClusterOperator reconciliation failed (attempt %d), retrying. ", o.failedReconcileAttempts+1)
-	if o.failedReconcileAttempts >= 2 {
-		// Only update the ClusterOperator status after 3 retries have been attempted to avoid flapping status.
-		klog.Warningf("Updating ClusterOperator status to failed after %d attempts.", o.failedReconcileAttempts+1)
+func (o *Operator) reportFailed(ctx context.Context, report runReport) {
+	report.isDegraded()
```

Comment:
not needed?

Reply:
You are right :). Happened during our pair programming session :D